src/lib/light.ts

```diff
@@ -1,14 +1,14 @@
 import { generateRandomHex } from "./utils.js";
 
 import axios, { AxiosInstance, AxiosResponse } from "axios";
-import FetchWrapper, { FetchResponse } from "./fetchwrapper.js";
+import FetchWrapper, { FetchResponse } from "./fetchwrapper";
 import delay from "delay";
 // dynamically import udp for compatibility with browser
 // import * as udp from "node:dgram";
 
-import { Led } from "./led.js";
-import { Frame } from "./frame.js";
-import { Movie } from "./movie.js";
+import { Led } from "./led";
+import { Frame } from "./frame";
+import { Movie } from "./movie";
```

Comment:
Removing the `.js` extension throughout fixed some module resolution quirks in VSCode for me